README.md

```diff
@@ -32,7 +23,27 @@ Your task is to create a React element that represents _you_ (or a close approxi
 </div>
 ```
 
+To start the application type this in the terminal:
+
+```
+npm install
+npm start
+```
+
+To run the test suite type this in the terminal: 
+
+```
+npm test
+```
+
 Feel free to render out the component in the DOM using `ReactDOM.render()`.
 
+## NOTE!! 
```

Comment:
Instead of this note, what do you think about creating a little aside here to introduce the idea of using unique keys in React?